client/my-sites/checkout/checkout/index.jsx

```diff
@@ -306,13 +306,25 @@ export class Checkout extends React.Component {
 
 		if (
 			( startsWith( this.props.product, 'jetpack_backup' ) ||
-				startsWith( this.props.product, 'jetpack_search' ) ||
 				startsWith( this.props.product, 'jetpack_scan' ) ) &&
 			isJetpackNotAtomic
 		) {
 			cartItem = jetpackProductItem( this.props.product );
 		}
 
+		if (
+			startsWith( this.props.product, 'jetpack_search' ) ||
+			startsWith( this.props.product, 'wpcom_search' )
+		) {
+			if ( isJetpackNotAtomic ) {
+				cartItem = jetpackProductItem( 'jetpack_search' );
```

Comment:
I think this may break purchasing the monthly plan for normal jetpack sites correct?

Reply:
addressed